zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/MetricFromDecisionEvaluationHandler.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.handlers;
+
+import static io.camunda.exporter.utils.ExporterUtil.tenantOrDefault;
+
+import io.camunda.exporter.store.BatchRequest;
+import io.camunda.webapps.schema.entities.MetricEntity;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.value.DecisionEvaluationRecordValue;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MetricFromDecisionEvaluationHandler
+    implements ExportHandler<MetricEntity, DecisionEvaluationRecordValue> {
+
+  protected static final String EVENT_DECISION_INSTANCE_EVALUATED =
+      "EVENT_DECISION_INSTANCE_EVALUATED";
+  private static final String ID_PATTERN = "%d-%d";
+
+  private final String indexName;
+
+  public MetricFromDecisionEvaluationHandler(final String indexName) {
+    this.indexName = indexName;
+  }
+
+  @Override
+  public ValueType getHandledValueType() {
+    return ValueType.DECISION_EVALUATION;
+  }
+
+  @Override
+  public Class<MetricEntity> getEntityType() {
+    return MetricEntity.class;
+  }
+
+  @Override
+  public boolean handlesRecord(final Record<DecisionEvaluationRecordValue> record) {
+    return true;
```

Comment:
❓ do we really want to handle all intents here? or only `EVALUATED`?

Reply:
Judging from the previous [implementation](https://github.com/camunda/camunda/blob/stable/8.6/operate/importer-8_6/src/main/java/io/camunda/operate/zeebeimport/processors/DecisionEvaluationZeebeRecordProcessor.java) I think we want to consume both `EVALUATED` and `FAILED` intents.

Reply:
that's interesting since the event name is EVENT_DECISION_INSTANCE_`EVALUATED`

Reply:
I think from business side even when execution failed decision instance was still technically executed and we should count this. We can double check with PM of course, but since it was implemented like this from the beginning, I would rather say that metric name is not fully correct. 
But again we can't change it as it's a part of contract and the metric is being documented and used this way 🤷

Reply:
can we then at least add another tag to the metric to separate evaluated than failed?

Reply:
We could theoretically split these events in two and also change the usage metrics query, but I'm not entirely sure where that expands to and what it could affect.